### PR TITLE
Add tvos build support

### PR DESCRIPTION
**Pull request by KazuCocoa**
- Bump lib version to 1.0.6 ( ref: https://github.com/appium/YYCache/releases )
- Add tvOS build support
    - I could build for tvOS and that worked on tvOS support. I tested with Xcode 10.1 and 10.2-beta
        - https://github.com/KazuCocoa/WebDriverAgent/tree/km/tvos
- Change deployment target from 8.0 to 9.0 following our support version

```
$ carthage update
*** Fetching RoutingHTTPServer
*** Fetching YYCache
*** Fetching CocoaAsyncSocket
*** Checking out YYCache at "d675f2fa837b74b05613b57a0432a188434abe73"
*** Checking out CocoaAsyncSocket at "7.6.3"
*** Checking out RoutingHTTPServer at "v1.0.2"
*** xcodebuild output can be found in /var/folders/34/2222sh8n27d6rcp7jqlkw8km0000gn/T/carthage-xcodebuild.8h0BVH.log
*** Downloading CocoaAsyncSocket.framework binary at "Version 7.6.3"
*** Building scheme "RoutingHTTPServer" in RoutingHTTPServer.xcodeproj
*** Building scheme "RoutingHTTPServer tvOS" in RoutingHTTPServer.xcodeproj
*** Building scheme "RoutingHTTPServer iOS" in RoutingHTTPServer.xcodeproj
*** Building scheme "YYCache iOS" in YYCache.xcodeproj
*** Building scheme "YYCache tvOS" in YYCache.xcodeproj
```

After this merge, we need 1.0.6 tag https://github.com/appium/YYCache/releases 
I have no permission to add the tag.

So, could someone create a new tag?)

### CODE REVIEW

File: Framework/Info.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>CFBundlePackageType</key>
 	<string>FMWK</string>
 	<key>CFBundleShortVersionString</key>
-	<string>1.0.4</string>
+	<string>1.0.6</string>
```
Comment (mykola-mokhnach) [inline Framework/Info.plist]: I'd rather bump the minor version number

File: Framework/Info.plist [hunk 2]
```diff
@@ -15,7 +15,7 @@
...
 	<key>CFBundleVersion</key>
 	<string>$(CURRENT_PROJECT_VERSION)</string>
```
Comment (mykola-mokhnach) [inline Framework/Info.plist:20]: should this one also be updated?

**Comment by mykola-mokhnach** [comment]
It would be cool if we could add basic travis config to this project to just check if commits are compilable

File: Framework/Info.plist (hunk 2 above)
Comment (KazuCocoa) [reply Framework/Info.plist:20]: No, I think.
I grepped the word. It will be _Build_ number.
_Version_ number is enough to bump the lib version.

**Comment by KazuCocoa** [comment]
Do you have permission to enable Travis?

**Comment by mykola-mokhnach** [comment]
@KazuCocoa Travis has been enabled

**Comment by KazuCocoa** [comment]
✅ 
`"YYCache iOS` and `"YYCache tvOS` schemes worked

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**